jest/setupFile.js

```diff
@@ -75,33 +75,3 @@ jest.mock('@react-navigation/stack', () => {
     },
   };
 });
-
```

Comment:
💯 , this was bothering me so much, thanks for pulling it out of the global context.